src/Validation/IndisposableValidation.php

```diff
@@ -0,0 +1,30 @@
+<?php
+
+namespace Propaganistas\LaravelDisposableEmail\Validation;
+
+use Propaganistas\LaravelDisposableEmail\Facades\Indisposable;
+
+class IndisposableValidation
+{
+
+    /**
+     * Default error message.
+     *
+     * @var string
+     */
+    static $errorMessage = 'Disposable email addresses are not allowed.';
```

Comment:
Explicit property visibility. Better make it protected as well.

Reply:
The `$errorMessage` property is used by the service provider for assigning a default validation error message.

> **src/DisposableEmailServiceProvider.php:**
```php
 $this->app['validator']->extend(
    'indisposable',
    IndisposableValidation::class . '@validate',
    IndisposableValidation::$errorMessage
);
```

Would you like me to instead remove the property and assign the error message inline with the validator extension?